src/main/java/address/util/OrderedList.java

```diff
@@ -0,0 +1,73 @@
+package address.util;
+
+import address.model.datatypes.person.ReadOnlyViewablePerson;
+import com.sun.javafx.collections.SourceAdapterChange;
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.TransformationList;
+
+/**
+ *
+ */
+public class OrderedList<T> extends TransformationList<T, T> {
+
+
+    private ObservableList<T> orderedList;
+
+    /**
+     * Creates a new Transformation list wrapped around the source list.
+     *
+     * @param source the wrapped list
+     */
+    public OrderedList(ObservableList<T> source) {
+        super(source);
+        orderedList = FXCollections.observableArrayList();
```

Comment:
The list will be updated in the sourceChanged method. at first the source is empty also.